packages/react/src/QuestionnaireForm.tsx

```diff
@@ -463,3 +483,16 @@ function isDropDownChoice(item: QuestionnaireItem): boolean {
       e.valueCodeableConcept?.coding?.[0]?.code === 'drop-down'
   );
 }
+
+function isQuestionEnabled(item: QuestionnaireItem, answers: Record<string, QuestionnaireResponseItemAnswer>): boolean {
+  if (!item.enableWhen) {
+    return true;
+  }
+  const expectedAnswer = getTypedPropertyValue(
+    { type: 'QuestionnaireItemEnableWhen', value: item.enableWhen[0] },
```

Comment:
The spec says that we should treat multiple conditions as "ORs". 
https://www.hl7.org/fhir/questionnaire-definitions.html#Questionnaire.item.enableWhen

I think it would be good just to implement that functionality now, so we don't have to worry about it later

Reply:
Hm, the spec contradicts itself.  It also says "Rule: If there are more than one enableWhen, enableBehavior must be specified"